android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'mapeo'
+include ':@react-native-community_checkbox'
+project(':@react-native-community_checkbox').projectDir = new File(rootProject.projectDir, '../node_modules/@react-native-community/checkbox/android')
```

Comment:
is this necessary? i think react native autolinking should handle this automatically ([link](https://github.com/react-native-checkbox/react-native-checkbox#mostly-automatic-installation))

Reply:
So I didnt add it, it was added though the autolinking command

Reply:
ah gotcha nvm then